### ISSUE TITLE
Project example

### ISSUE THREAD

**Issue by kopax** [jclagache/spring-data-mybatis#5]
Hi, I am trying to implement this in my project.

I was previously using services, but for some reasons it sounds easier to have repository to get best part of spring for creating a rest API. (I wan't to implement spring-hateoas and with repository most of the stuff get autoconfigured)

I saw your project has some solicitation and was updated a month ago.

I am trying to implement it but I feel a bit lost with the errors I have in my current system. 

It would be nice to have a working example repository.

**Comment by kopax** [comment]
Close as this will be done after #8
